packages/documentation-site/patternfly-docs/content/get-started/upgrade.md

```diff
@@ -9,25 +9,32 @@ import { Alert } from '@patternfly/react-core';
+PatternFly 6 introduces exciting new features and functionality, as described in [our release highlights](/get-started/release-highlights). To support these new offerings, we've introduced a long list of changes, the details of which can be found in our [major release notes](/get-started/upgrade/release-notes). 
 
-This guide outlines the requirements for upgrading your codebase to PatternFly 6 and provides additional resources to support your migration efforts.  
+This guide outlines the steps required to upgrade your codebase to PatternFly 6 and provides additional resources to support your migration efforts.  
 
 ## Get help 
 
 If you need support as you upgrade, the PatternFly team is here to help! Reach out to us on [Slack](https://join.slack.com/t/patternfly/shared_invite/zt-1npmqswgk-bF2R1E2rglV8jz5DNTezMQ) or ask a question on our [GitHub discussion board](https://github.com/orgs/patternfly/discussions). We'll always do our best to answer your questions and connect you with the right people quickly. 
 
 ## How to upgrade
 
-When you upgrade your product to PatternFly 6, several breaking changes will likely be introduced to your product’s codebase. 
+When you upgrade your product to PatternFly 6, several breaking changes will likely be introduced to your product’s codebase. These steps walk you through the migration process and connect you with resources that will help you address these breaking changes.
 
-Where we can, we offer a suite of codemods to simplify and streamline your upgrade process. Instead of requiring you to manually identify all errors and issues in your codebase, you can run our codemods to quickly identify and fix major issues. Keep in mind that some changes will still require manual intervention, but our codemods can automatically fix a large amount of issues and flag any issues that do require manual work.
+### 1. Remove all CSS overrides
+At the start of your upgrade process, remove all of your CSS overrides and see how things look. Then, add back any necessary customizations. Since there often isn't a 1:1 equivalent for every PatternFly 5 style in PatternFly 6, your previous overrides may not even be relevant. 
 
-**Note:** Running codemods after making manual changes will lead to even more manual intervention and cleanup. Because of this, we strongly recommend running codemods _before_ making any manual changes to your codebase.
+As much as possible, we recommend removing CSS overrides so that your product upgrade experience will be smoother for future releases. 
+
+If you decide to keep any overrides, be sure to [follow step 3](#3-review-and-update-variable-and-class-names) to ensure they're up to date.
```

Comment:
Not necessarily a suggestion - for an anchor link like this it's fine as-is - but if you wanted to try and have the link be descriptive enough on its own for what it does, this would be one way

```suggestion
If you decide to keep any overrides, be sure to follow step 3 below and [review and update variable and class names](#3-review-and-update-variable-and-class-names) to ensure they're up to date.
```